test/integration/test-example-validation.js

```diff
@@ -30,7 +30,7 @@ if (!window.validatorLoad) {
   })();
 }
 
-describe.configure().retryOnSaucelabs().run('example', function() {
+describe.skip('example', function() {
```

Comment:
`gulp test` uses `testPaths` which is `test/**/*`. We definitely want to keep running integration tests on Travis with latest Chrome, so maybe the job can be renamed from `presubmit and unit tests` to something like `presubmit and all tests on latest Chrome` and the `integration tests` job can become `integrations tests on SauceLabs`